core/templates/dev/head/pages/exploration_editor/editor_tab/InteractionDetailsCacheService.js

```diff
@@ -0,0 +1,44 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** 
+ * @fileoverview service to provide state-specific cache for interaction
+ * details.It stores customization argscorresponding to an interaction id so
```

Comment:
@GaganSuneja please could you proofread the PR prior to submission? There should be spaces after "." and also between "args" and "corresponding".